opentoken/_utils.py

```diff
@@ -125,6 +125,16 @@ def otk_str_to_ordered_dict(otk_str):
         OrderedDict: OrderedDict representation of the token.
 
     """
-    items = otk_str.split("\n")
-    pairs = [tuple(line.split("=")) for line in items]
-    return OrderedDict(pairs)
+    items = otk_str.rstrip().split("\n")
+    pairs = [tuple(filter(None,re.split(r"=(.+)?", line))) for line in items]
+
+    data = dict()
```

Comment:
Thanks for the update, have been bad about notification follow ups. Though the opentoken specs don't specify that data be ordered in a specific format, I prefer to OrderedDict to ensure uniqueness.